pkg/httperrors/httperrors.go

```diff
@@ -39,7 +39,7 @@ var codes = map[int]string{
 	500: "Internal Server Error",
 	501: "Not Implemented",
 	502: "Bad Gateway",
-	503: "Service Unavailable",
+	503: "AppService Unavailable",
```

Comment:
This should be a generic default message. This affects areas of the server. It should remain as the default and if we want to at some point return "AppService Unavailable" it can be supplied as an argument.

https://www.lifewire.com/503-service-unavailable-explained-2622940